imqsauth/http.go

```diff
@@ -224,7 +224,7 @@ func (x *ImqsCentral) IsAdmin(r *http.Request) (bool, error) {
 func (x *ImqsCentral) makeAbsoluteUrl(relativeUrl string) (string, error) {
 	hostname := x.Config.GetHostname()
 	if hostname == "" {
-		return "", fmt.Errorf("'hostname' is not configured on this server")
+		return "", fmt.Errorf("'hostname environment variable' is not set on this server")
```

Comment:
Maybe also indicate the hostname variable name,`IMQS_HOSTNAME_URL` as part of the error message.